frontend/pages/DetailedViewPage.js

```diff
@@ -40,7 +63,6 @@ export default class DetailedViewPage extends Component {
                 </View>
```

Comment:
In line 61, it might be better to refactor the inline styling into the stylesSheet

Reply:
Good catch, will do!